class.jetpack-options.php

```diff
@@ -120,6 +120,24 @@ public static function get_option( $name, $default = false ) {
 		return $default;
 	}
 
+	/**
+	 * Returns the requested option, and ensures it's autoloaded in the future.
+	 * This does _not_ adjust the prefix in any way (does not prefix jetpack_%)
+	 *
+	 * @param string $name Option name
+	 * @param mixed $default (optional)
+	 */
+	public static function get_option_and_ensure_autoload( $name, $default ) {
+		$value = get_option( $name );
+		
+		if ( $value === false ) {
+			update_option( $name, $default, null, true );
```

Comment:
This could lend into calling `update_option` many times If the default for what we are trying to request is `false`.

Maybe we should have a routine on plugin activation/upgrade that's sets defaults values for all Jetpack options?

Note: just saw that `update_option` doesn't add the option to the database if the value is `false`, I think we will need to use `add_option` for this.